package.json

```diff
@@ -36,7 +36,7 @@
     "bcrypt": "^0.8.5",
     "bluebird": "^3.3.1",
     "body-parser": "^1.14.1",
-    "common-errors": "^0.5.3",
+    "common-errors": "github:lackeyjb/node-common-errors",
```

Comment:
Yeah I forked this package, and made a few modifications. Mainly, I changed a middleware file that we're using as our error handler to send the error as JSON, and I changed the JSON object to send as so:
```
{
  "errorType": "some error type",
  "message": "some error message",
  "status": "some HTTP status"
}
```
I also added a package called [pretty-error](https://www.npmjs.com/package/pretty-error) that formats stack traces a little nice.